corehq/apps/registration/views.py

```diff
@@ -89,7 +89,7 @@ def register_user(request):
 
         context.update({
             'form': form,
-            'legacy_password': getattr(settings, 'ENABLE_DRACONIAN_SECURITY_FEATURES', False),
+            'show_password_feedback': not getattr(settings, 'ENABLE_DRACONIAN_SECURITY_FEATURES', False),
```

Comment:
Clarified offline, logic is fine but I would also exclude the `zxcvbn` lib along with `password.js`. There are also 3 other places where this frontend validation is done which should receive the same treatment:
* password reset
* change password
* accept invite